### PR TITLE
Replace var with const

### PR DESCRIPTION
**Pull request by tanabee**
I replaced `var` with `const` in the document below.
https://reactjs.org/docs/hooks-state.html#tip-what-do-square-brackets-mean

Reference: https://github.com/reactjs/reactjs.org/blob/master/CONTRIBUTING.md#use-const-where-possible-otherwise-let-dont-use-var

### CODE REVIEW

File: content/docs/hooks-state.md [hunk 1]
```diff
@@ -235,9 +235,9 @@ The names on the left aren't a part of the React API. You can name your own stat
 This JavaScript syntax is called ["array destructuring"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring). It means that we're making two new variables `fruit` and `setFruit`, where `fruit` is set to the first value returned by `useState`, and `setFruit` is the second. It is equivalent to this code:
 
 ```js
-  var fruitStateVariable = useState('banana'); // Returns a pair
-  var fruit = fruitStateVariable[0]; // First item in a pair
-  var setFruit = fruitStateVariable[1]; // Second item in a pair
```
Comment (vnoitkumar) [inline content/docs/hooks-state.md]: Hahah

**Comment by alexkrolick** [comment]
I think this was written this way because people unfamiliar with array destructuring are probably not familiar with the latest JS syntax

**Comment by sophiebits** [comment]
Agreed; it was intended to be written this way. Thank you for contributing though!

**Comment by tanabee** [comment]
I understand. Thank you for your feedback!

**Pull request closed without merging**